File: 448-find-all-numbers-disappeared-in-an-array/448-find-all-numbers-disappeared-in-an-array.ts
function findDisappearedNumbers(nums1: number[]): number[] {
    const arr = [];
    const nums = new Set(nums1);
    for(let i = 1; i <= nums1.length; i++) {
        if(!nums.has(i)) arr.push(i);
    }
    return arr;
};

File: 448-find-all-numbers-disappeared-in-an-array/448-find-all-numbers-disappeared-in-an-array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        numset = set(nums)
        output = []
        
        for i in range(1, len(nums)+1):
            if not i in numset:
                output.append(i)
        return output
        
        